CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 # Setup third-party dependencies
 include(conan)
+conan_add_remote(NAME bincrafters
+    URL https://api.bintray.com/conan/bincrafters/public-conan
```

Comment:
that's a deprecated url. check the [docs](https://bincrafters.readthedocs.io/en/latest/using_packages.html#adding-the-bincrafters-repository-as-a-conan-remote) and how @q4a [integrated sdl2](https://github.com/q4a/storm-engine/commit/0a78c174129c2b265324e5b155766b80bb3076cd#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR10)

Reply:
Not on binary? :O

Reply:
what do you mean?

Reply:
T9 :( I meant bintray. I believe it's not deprecated but rather a release repo link...

Reply:
Ok I see it's getting closed soon